src/app/store/machine/utils/hooks.ts

```diff
@@ -61,7 +71,7 @@ export const useGetMachine = (
   }, [dispatch, id, previousId]);
   // TODO: clean up the previous request if the id changes or the component is unmounted:
   // https://github.com/canonical-web-and-design/app-tribe/issues/1151
-  return requestId.current;
+  return { machine, loading, loaded };
```

Comment:
I was wondering about doing this when I created the hook, it's certainly much easier to use, but wasn't sure how it'd look for the `useFetchMachines` as I'm not sure if we know what selector the hook will need to use, but I suppose we can always revisit this if we end up with a slightly different API.